pkg/network/encoding/format.go

```diff
@@ -88,9 +89,18 @@ func FormatConnection(
 		c.HttpAggregations, _ = proto.Marshal(httpStats)
 	}
 
+	httpStats2, staticHTTP2Tags, dynamicHTTP2Tags := http2Encoder.GetHTTP2AggregationsAndTags(conn)
```

Comment:
do you have dynamic tags? do you have tags?

Reply:
removed